control-plane/csi-driver/src/bin/controller/client.rs

```diff
@@ -11,6 +11,7 @@ use stor_port::types::v0::openapi::{
 
 use anyhow::{anyhow, Result};
 use once_cell::sync::OnceCell;
+use stor_port::types::v0::openapi::models::VolumeGroup;
```

Comment:
move this above together with the other stor_port uses.

Reply:
Done.